[Issue #807] Ner'zhul keeps Orc trait

pedromdpereira opened the pull request:
## Changelog:
- Orc racial trait was being changed when Ner'zhul is lich king. This version removed that instruction.
- Close #807

![image](https://user-images.githubusercontent.com/35965798/148702992-a6a87ae2-0071-41c7-9370-fb70f51ec1bb.png)

pedromdpereira commented:
It's my first pull request. Anything I can improve, please let me know :)

zumbak04 commented:
Allow changes to a pull request, please
https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork

zumbak04 commented on history/characters/10000_orc.txt:
```diff
@@ -334,7 +334,6 @@
 		culture=scourge religion=death_god
 		trait = being_undead
-		remove_trait = creature_orc
 		trait = creature_lich
```
Ner'zhul shouldn't have an orc trait at this point. That's not what we try to achieve.
To make the race change working, there's fix that looks like:
```suggestion
		effect = { add_trait_force_tooltip = creature_lich }
```
Have to notice it's not a unique issue. You may find more by looking for the `remove_trait = creature_` pattern in `history\characters` folder.

pedromdpereira commented:
Changed the code based on your sugestion. Found another examples of that patterns and fixed them too.
Examples:
![image](https://user-images.githubusercontent.com/35965798/148945916-580ff7b3-f335-4013-af5c-78294272c7b8.png)
![image](https://user-images.githubusercontent.com/35965798/148945949-b418bc4e-ca2f-4354-a3bd-ec3f9453a56e.png)
![image](https://user-images.githubusercontent.com/35965798/148946037-51025538-ddb3-44ae-8286-d792e78af912.png)

zumbak04 commented on history/characters/10000_orc.txt:
```diff
@@ -336,6 +336,7 @@
 		trait = being_undead
```
First death knights don't change race traits.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/149533464-a0c3310f-9cb2-446e-871e-06e419ef388f.png)

</details>

pedromdpereira replied:
Changed this for Teren'gor and Ragnok
![image](https://user-images.githubusercontent.com/35965798/149587543-7eedad43-2983-41f9-b3c1-20bf204a6b56.png)

zumbak04 commented on history/characters/10000_orc.txt:
```diff
@@ -336,6 +336,7 @@
 		trait = being_undead
 		remove_trait = creature_orc
```
Vashj don't change race trait.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/149533719-a14ad90e-9c38-4572-a95d-c9f4f46da121.png)

</details>

pedromdpereira replied:
Also fixed
![image](https://user-images.githubusercontent.com/35965798/149587627-c25052f9-c26f-4fe2-a5b7-6f25b80396a0.png)

zumbak04 commented on history/characters/110000_scourge.txt:
```diff
@@ -15,6 +15,7 @@
 		remove_trait=creature_orc
 		trait=creature_lich
 		trait=being_undead
+		effect = { add_trait_force_tooltip = creature_lich }
```
There's no need to keep `trait=` line around.
P.S. The same with other `trait` that were replaced by `add_trait_force_tooltip`.
```suggestion
		trait=being_undead
		effect = { add_trait_force_tooltip = creature_lich }
```

pedromdpereira replied:
I cleaned up the code a bit based on your sugestion.

zumbak04 reviewed (commented):
Hello!
Left a few more comments.

zumbak04 commented on history/characters/10000_orc.txt:
```diff
@@ -335,7 +335,7 @@
 		trait = being_undead
 		remove_trait = creature_orc
-		trait = creature_lich
+		effect = { add_trait_force_tooltip = creature_lich }
```
I'm sorry to bother you with such a thing since everything works.
But I found out it may work like this:
```suggestion
		add_trait = creature_lich
```
This is much clearer.
I would do it myself, but I'm not allowed to push on this branch.

pedromdpereira replied:
No problem, I also agree with that approach! Worked perfectly
![image](https://user-images.githubusercontent.com/35965798/149621891-23d2757c-b191-414f-a4f2-526c3806a42c.png)

zumbak04 reviewed (approved):
LGTM!
